src/SprykerEco/Yves/Adyen/Form/AbstractSubForm.php

```diff
@@ -25,10 +25,18 @@ public function getProviderName(): string
     }
 
     /**
+     * @param array|null $options
+     *
      * @return \Symfony\Component\Validator\Constraint
      */
-    protected function createNotBlankConstraint(): Constraint
+    protected function createNotBlankConstraint(?array $options): Constraint
     {
-        return new NotBlank(['groups' => $this->getPropertyPath()]);
+        $nonBlank = new NotBlank(['groups' => $this->getPropertyPath()]);
+
+        if ($options !== null && is_string($options['message'])) {
```

Comment:
Are you sure the `$options`  being not null will definitely mean that `'message'` key is present in it? Please check the key exists before accessing.